core/domain/platform_feature_services_test.py

```diff
@@ -81,15 +83,15 @@ def setUp(self):
                             'conditions': [
                                 [
                                     '=',
-                                    platform_parameter_domain.SERVER_MODES.dev
+                                    SERVER_MODES.dev.value
```

Comment:
You can do `['=', SERVER_MODES.dev.value],`

Reply:
ditto in other places

Reply:
done